api.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import path from 'path';
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import { getXls } from './getXls.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.get('/transactions', async (req, res) => {
+  try {
+    await getXls();
+
+    const downloadPath = path.resolve(__dirname, 'downloads');
+    const jsonFilePath = path.join(downloadPath, 'data.json');
+    const jsonData = fs.readFileSync(jsonFilePath, 'utf-8');
```

Comment:
no...

`getXls` fn will return the raw json after processing the xls, so, no need to read the file system anymore